VimeoNetworking/Sources/AuthenticationController.swift

```diff
@@ -148,10 +148,6 @@ final public class AuthenticationController
 
             try self.setClientAccount(with: loadedAccount)
         }
-        else
```

Comment:
Why are we removing this code?  It seems like this prints a warning that there was a failure when attempting to load the stored account.  If anything, this else statement should probably throw an error instead of just printing to the console.

Reply:
+1 Can we throw if either `try self.accountStore.loadAccount(ofType: accountType)` or `try self.setClientAccount(with: loadedAccount)` fails?

Reply:
@mikew-personal @nicolelehrer - I ticketed this work to surface this issue to the client.